src/config/colors.toml

```diff
@@ -34,21 +36,21 @@
 "working_copy divergent" =  "bright red"
 "working_copy divergent change_id"="bright red"
 "working_copy conflict" = "bright red"
-"working_copy description" = "bright white"
+"working_copy description" = "bright normal"
```

Comment:
I don't think this has any effect, so we can just leave these out instead. See https://github.com/martinvonz/jj/blob/aef080191726d3ffc823a98930e3a1bafb84a62a/src/formatter.rs#L325-L345

Also, it does make the working-copy commit stand out a bit less, but I guess we'll just have to accept that if we want to make the text readable on dark or white background.

Reply:
Fixed. I'm not quite sure what "diff file_header" is. I made it bold, but could remove that.

I agree about the working copy commit. It's fortunate you made it configurable.

Reply:
For completeness, it's possible to implement "bright normal" with the old bright escape sequence. It would always be bold. It might also lead to some messy code, I'm not sure.

Reply:
I'll merge this as is. We can always remove the "diff file_header" later and I certainly wouldn't put anything having to do with the (possibly crazy) "bright normal" idea in this PR.

Reply:
The `diff file_header` is in `jj diff --git` output. I think it's fine that it's just bold.

> it's possible to implement "bright normal" with the old bright escape sequence.

Isn't that what you get with `{bold = true}`? That results in just ANSI code 1 for bold font. If you want to also get bright color (even on terminals that don't do bold=>bright), I think you have to explicitly pick one of the bright colors, and I don't think there's a "bright normal" color to set, is there?